support-frontend/test/selenium/util/PostDeployTestUser.scala

```diff
@@ -24,3 +28,31 @@ class PostDeployTestUser(driverConfig: DriverConfig) extends TestUser {
   val username = testUsers.generate()
   addTestUserCookies(username)
 }
+
+class PostDeployTestUserSubs(driverConfig: DriverConfig) extends TestUser {
+
+  private val client = new OkHttpClient()
+  private val requestToIdapi = new Request.Builder()
+    .url(Config.idapiNewTestUserUrl)
+    .addHeader(Config.idapiClientAccessTokenName, Config.idapiClientAccessTokenSecret)
+    .post(RequestBody.create(MediaType.parse("application/json"), "{}"))
+    .build()
+  private val response = client.newCall(requestToIdapi).execute()
+  private val responseBodyAsJson = parse(response.body().string()).getOrElse(Json.Null)
```

Comment:
It would be good to deserialise this response to a case class using [Circe](https://github.com/circe/circe). There are loads of examples of this in the code base, just ask me on chat if you'd like any pointers

Reply:
Got it, thanks! Addressed in the new commit